src/sentry/static/sentry/app/views/app.jsx

```diff
@@ -151,6 +152,7 @@ const App = createReactClass({
           <Alerts className="messages-container" />
           <Indicators className="indicators-container" />
           {this.props.children}
+          {ConfigStore.get('features').has('assistant') && <AssistantHelper />}
```

Comment:
Since `assistant` is defined on the organization model, shouldn't this come from the org data? I assume at this point we're also not necessarily in the context of a single org yet so that may not be ideal. Are users supposed to see this before they have created their first org?

Reply:
The reason for scoping to organization is so we can release it to early adopters (an org concept), plus uniformity (odd if some members see the assistant and others don't). But we also want it to be available outside of the org context, e.g. account settings page. You're right this doesn't work. I guess I could scope it to user and if we end up wanting to test it on early adopters, in django we can query all the orgs of the user and gate them in if any of them are early adopters.

Reply:
Fixed now.